src/mono/browser/runtime/invoke-js.ts

```diff
@@ -24,6 +24,7 @@ export const js_import_wrapper_by_fn_handle: Function[] = <any>[null];// 0th slo
 export function mono_wasm_bind_js_import_ST (signature: JSFunctionSignature): VoidPtr {
     if (WasmEnableThreads) return VoidPtrNull;
     assert_js_interop();
+    signature = ((signature as any) >>> 0) as any;
```

Comment:
can we make a helper function in memory for this? i.e. `fixupPointer` or something